resources/js/lazyLoadModuleOnIdFound.ts

```diff
@@ -5,15 +5,19 @@
  * @param {Object} options - Configuration options for the lazy loading process.
  * @param {string} options.elementId - The `id` attribute value of the DOM element that triggers the import when found.
  * @param {string} options.codeFileName - The name of the file in the dynamic folder to lazy load.
- * @param {string} options.moduleName - The name under which the imported module will be attached to the window object.
+ * @param {string} options.moduleNameToAttachToWindow - The name under which the imported module will be attached to the window object.
  */
-export const lazyLoadModuleOnIdFound = (options: { elementId: string; codeFileName: string; moduleName: string }) => {
-  const { elementId, codeFileName, moduleName } = options;
+export const lazyLoadModuleOnIdFound = (options: {
+  elementId: string;
+  codeFileName: string;
+  moduleNameToAttachToWindow: string;
```

Comment:
Renamed this to hopefully make it more clear what this `lazyLoadModuleOnIdFound` function is doing.